simcon_project/conversation_templates/tests/test_template_management.py

```diff
@@ -1,10 +1,11 @@
 from django.test import TestCase, RequestFactory
 from django.utils import timezone
 from django.shortcuts import reverse
-from users.models import Researcher
-from ..models import TemplateFolder, ConversationTemplate
+from datetime import date
+from users.models import Researcher, Student, Assignment, SubjectLabel
+from ..models import *
 from ..forms import FolderCreationForm
-from ..views.template_management import route_to_current_folder, remove_template, FolderEditView
+from ..views.template_management import route_to_current_folder, FolderEditView
```

Comment:
Route to current folder, should not be snake case. Not sure if this was you or before

Reply:
remove_template shouldn't be snake case either then right?